### ISSUE TITLE
Question about train()

### ISSUE THREAD

**Issue by ZN-ning** [princeton-vl/CER-MVS#2]
Great job!
Why do you use model.eval() instead of model.train() in the train function?

**Comment by mazeyu** [comment]
looks like this is an accidental typo. Will test how much difference it makes and get back to you

**Comment by mazeyu** [comment]
Hi, I tested and confirmed eval/train does not affect the current code as we don't use batch normalization, just instance normalization. The reason why it was "eval" is that in early experiments we found it more stable (possibly with other normalization layers). But still I changed the code to the normal setting so as to not confuse people.
